packages/iov-keycontrol/src/hdpaths.ts

```diff
@@ -1,15 +1,28 @@
 import { Slip10RawIndex } from "@iov/crypto";
 
 export class HdPaths {
+  public static readonly purposes = {
+    bip44: 44,
+    iov: 4804438,
+    iovFaucet: 1229936198,
+  };
+  /**
+   * Coin types as registered at
+   * https://github.com/satoshilabs/slips/blob/master/slip-0044.md
+   */
+  public static readonly coinTypes = {
+    eth: 60,
+    iov: 234,
+  };
+
   /**
    * IOV's SimpleAddress derivation path
    *
-   * @see https://github.com/iov-one/iov-core/blob/v0.6.1/docs/KeyBase.md#simple-addresses
+   * @see https://github.com/iov-one/iov-core/blob/master/docs/address-derivation-v1.md#simple-addresses-deprecated
```

Comment:
Can we use a specific tag here instead of master to ensure the link works (even if it gets outdated over time)